presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -49,6 +51,23 @@ public BaseJdbcConfig setConnectionUrl(String connectionUrl)
         return this;
     }
 
+    @AssertTrue
+    public boolean isConnectionUrlValid()
+    {
+        String connectionUrl = getConnectionUrl();
+        if (!connectionUrl.startsWith("jdbc:")) {
+            return false;
```

Comment:
This should be done with a regex validator

Reply:
Why? Please elaborate